### DISCUSSION TITLE
Is it possible to add 'Change Icon' option to multi selected items ?

### DISCUSSION THREAD

**Discussion by aniketgm** [FlorianWoelki/obsidian-icon-folder#620] [Q&A]
Currently, when we multi-select and 'right click' we don't see the option 'Change Icon'

So, the use-case is - I would like some folder as well as some files to have same icon. Now for me the folder are limited so I can select one-by-one -> right click and apply icon. 

However there are like 100s of files for which I would like to have same icons. Right clicking each one-by-one and applying the icon, might bring me close to my retirement age.

Before raising an issue, I wanted to know, if it's even possible to add an option to multi selected files.
If Yes, I can raise and issue for it. If No, then I have to plan my retirement plan soon before I start the Right clicking process.
Just kidding, but you get what I mean.

If this is not a iconize question, I apologize, but then I'm not sure, where this feature came from. Is it the main site ?

**Comment by FlorianWoelki** [comment]
That definitely makes sense, but it has not been implemented yet.
Feel free to create an issue for that! Thank you :)
